src/main/java/webserver/HttpResponse.java

```diff
@@ -0,0 +1,43 @@
+package webserver;
+
+import java.io.DataOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HttpResponse {
+    private static final Logger log = LoggerFactory.getLogger(HttpResponse.class);
+    private DataOutputStream dos;
+    private final String ROOT_PATH = "src/main/resources/templates";
+
+    public HttpResponse(DataOutputStream dos) {
+        this.dos = dos;
+    }
+
+    public void response200Header(int lengthOfBodyContent) {
```

Comment:
```suggestion
    public HttpResponse response200Header(int lengthOfBodyContent) {
```
이렇게 하면 체이닝을 사용할 수 있어서 깔끔할 것 같아요.

response.ok().body() ... 이렇게 사용할 수 있겠죠.